source/Coop/Mod/GameSync/Initializer.cs

```diff
@@ -29,6 +31,11 @@ public static void SetupSyncAfterLoad()
                 FiefSync.MakeManaged(town);
             }
 
+            //foreach (Town town in Town.AllTowns)
```

Comment:
Move this commented code to a different branch

Reply:
Done